src/chapters/objects-and-math/background.rst

```diff
@@ -0,0 +1,132 @@
+   :alt: Diagram showing that objects are a collection of key/value pairs.
+
+Initializing Objects
+--------------------
+
+.. index:: ! object literal
+
+When defining an object, we call the intialization an **object literal**.
+We will need three things to create an object: the name of the object, the keys, and their corresponding values.
+
+.. note::
+
+   Object literals use curly braces, ``{}``, to enclose the key/value pairs.
+
+Once we have these three things, we can write an object literal like so:
+
+.. sourcecode:: js
+   :linenos:
+
+   let objectName = {key1:value1, key2:value2, key3:value3, ... };
+
+If we have a lot of key/value pairs in our object, we can also put each one on a separate line!
+
+.. sourcecode:: js
+   :linenos:
+
+   let objectName = {
+       key1: value1,
+       key2: value2,
+       key3: value3,
+       .
+       .
+       .
+   };
+
+.. warning::
+
+   When putting the key/value pairs on a separate line, it is important to pay attention to spaces and tabs!
+   Incorrect spacing or tab usage can result in a bug.
+
+When defining an object, keep in mind that the keys can only be valid JavaScript strings.
+The values can be any of the data types that we have previously discussed.
+
+.. admonition:: Example
+
+   Let's say that we want to create a small program for a zoo.
+   We could create an object for storing the different data points about the animals in a zoo.
+   We start with our first tortoise. His name is Pete! He is an 85 year old, 919 lb Galapagos Tortoise, who prefers a diet of veggies.
+   Our object literal for all of this important data about Pete would be: 
+
+   .. sourcecode:: js
+      :linenos:
+
+      let tortoiseOne = {
+          species: "Galapagos Tortoise",
+          name: "Pete",
+          weight: 919,
+          age: 85,
+          diet: ["pumpkins", "lettuce", "cabbage"]
+      };
+
+
+Methods and Properties
+----------------------
+
+.. index:: ! property
+
+.. index:: method
+
+A **property** of an object is a key/value pair of an object.
+The property's name is the key and the property's value is the data assigned to that key.
+
+A **method** performs an action on the object, because it is a property that stores a function.
+
+.. admonition:: Example
+
+   In the case of Pete, our zoo's friendly Galapagos Tortoise, the object ``tortoiseOne`` has several properties for his species, name, weight, age, and diet.
+   If we wanted to add a method to our object, we might add a function that returns a helpful statement for the general public.
+
+   .. sourcecode:: js
+      :linenos:
+
+      let tortoiseOne = {
+          species: "Galapagos Tortoise",
+          name: "Pete",
+          weight: 919,
+          age: 85,
+          diet: ["pumpkins", "lettuce", "cabbage"],
+          sign: function() {
+              return this.name + " is a " + this.species;
+          }
+       };
+
+In the example above on line 8, we see a keyword which is new to us.
```

Comment:
I added a comma, but I wrote that because it is outside of the example admonition and I didn't want any ambiguity. Do you think it would be fine without the additional directive?

Reply:
In this case, I do think the line is fine without the extra directive.